dask_jobqueue/core.py

```diff
@@ -161,7 +168,8 @@ def job_file(self):
     def start_workers(self, n=1):
         """ Start workers and point them to our local scheduler """
         workers = []
-        for _ in range(n):
+        num_jobs = min(1, math.ceil(n / self.worker_processes))
```

Comment:
Why using `min` here? This would always lead to only one job started if I'm not mistaken.

Reply:
good point. I've removed this.